packages/expo-modules-core/android/src/androidTest/java/expo/modules/kotlin/jni/JSIAsyncFunctionsTest.kt

```diff
@@ -196,15 +197,15 @@ class JSIAsyncFunctionsTest {
     Truth.assertThat(exception.message).contains("java.lang.IllegalStateException")
   }
 
-//  @Test(expected = JavaScriptEvaluateException::class)
-//  fun should_throw_if_js_value_cannnot_be_passed() = withJSIInterop(
-//    inlineModule {
-//      Name("TestModule")
-//      AsyncFunction("f") { _: Int -> }
-//    }
-//  ) { methodQueue ->
-//    waitForAsyncFunction(methodQueue, "expo.modules.TestModule.f(Symbol())")
-//  }
+  @Test(expected = JavaScriptEvaluateException::class)
+  fun should_throw_if_js_value_cannot_be_passed() = withJSIInterop(
```

Comment:
```suggestion
  @Test(expected = JavaScriptEvaluateException::class)
  @Ignore
  fun should_throw_if_js_value_cannot_be_passed() = withJSIInterop(
```
This test won't pass, I'll take care of that later. You can ignore that for now ;)

Reply:
the issue is addressed by https://github.com/expo/expo/pull/19559/commits/b73afef258a86dc3daa3f3a422a83920e730e745, which is coming from jscexecutor